tests/ml/test_learn.py

```diff
@@ -14,4 +14,4 @@ def test_stack_arrays():
     os.remove("temp.npz")
     feature_names = "F1,F2,F3"
     result = stack_arrays(loaded, feature_names)
-    assert result.shape == (9, 2)
```

Comment:
ok now why is this broken?!?! I thought we tested and it should be 9,2 not 2,9

Reply:
Ick. Ok. change the test to be the right test and use the utils stack_arrays.
Should also change ml.learn to use the utils version as well as the other instances

Reply:
because the stack_array function is from the other duplicate code.

Reply:
I reverted the change because this test folder is for the ml module and it doesn't use utils.stack_arrays.

Reply:
def test_stack_arrays():
        data1 = np.random.random((3, 2))
        data2 = np.zeros((3, 2))
        data3 = np.ones((3, 2))
        np.savez("temp.npz", F1=data1, F2=data2, F3=data3)
        loaded = np.load("temp.npz")
        os.remove("temp.npz")
        feature_names = "F1,F2,F3"
        result = stack_arrays(loaded, feature_names)
>       assert result.shape == (9, 2)

Shouldn't the shape actually be (3, 6) since it is hstack?

Reply:
oh um let's think. what should be correct?
it's 3 samples x 2 dimensions, so that's like 3 samples of "image1[x,y]" "image2[x,y]" and "image3[x,y]"
and if we stack it all together, it should still be 3 samples x 6 dimensions, right? "x1,y1,x2,y2,x3,y3"
so yeah: you're right! it should be 3,6